### PR TITLE
894 Creating the ucgy_gbl_divider brick feature

### PR DESCRIPTION
**Pull request by robbdavis**
894 Creating the ucgy_gbl_divider brick feature

### CODE REVIEW

**Comment by robbdavis** [comment]
Thanks Navinder. Strange, I thought title fields were automatically required. I guess it's different for FPP's? Anyways good find, I've added the requirement through the form alter.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_divider/templates/fieldable-panels-pane--gbl_divider.tpl.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?php if ($logged_in && $is_admin) { ?><!-- Include file: <?php echo str_replace('-', '=', __FILE__); ?> --><?php } ?>
+
+<?php
+    // Set some variables.
+    // Gets the hex color and assigns it the corresponding css class.
+    $line_colour = '';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_divider/templates/fieldable-panels-pane--gbl_divider.tpl.php]: Commenting here to flag this for the future. Hard coded hex keys in template that will need to be resolved as some point. Nothing needs to change here Robb.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_divider/templates/fieldable-panels-pane--gbl_divider.tpl.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    if (isset($content['field_gbl_line_clour']['#items'][0]['rgb'])) {
+        $bg_hex = $content['field_gbl_line_clour']['#items'][0]['rgb'];
+        switch ($bg_hex) {
+            case '#662400':
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_divider/templates/fieldable-panels-pane--gbl_divider.tpl.php]: I know I've asked this before, but I'm going to ask again: 

Why are we duplicating this code all the time? Why isn't it centralized?

Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_divider/templates/fieldable-panels-pane--gbl_divider.tpl.php]: I don't know. Do you want me to write up a story for this?

Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_divider/templates/fieldable-panels-pane--gbl_divider.tpl.php]: Yeah, please separate story, medium priority. We gotta get this truly resolved. Make sure it goes to Jaela for review and fleshing out.

**Pull request merged**